verifiers/internal/gcb/keys/keys.go

```diff
@@ -0,0 +1,70 @@
+package keys
+
+import (
+	"crypto/ecdsa"
+	"crypto/x509"
+	"embed"
+	"encoding/pem"
+	"fmt"
+	"io/fs"
+	"path"
+
+	serrors "github.com/slsa-framework/slsa-verifier/errors"
+)
+
+//go:embed materials/*
+var publicKeys embed.FS
+
+type PublicKey struct {
+	Value  []byte
+	Region string
+	// TODO: key type and size
+}
+
+func PublicKeyNew(region string) (*PublicKey, error) {
+	content, err := fs.ReadFile(publicKeys, path.Join("materials", region+".key"))
+	if err != nil {
+		return nil, fmt.Errorf("%w: cannot read key materials", err)
+	}
+
+	return &PublicKey{
+		Value:  content,
+		Region: region,
+	}, nil
+}
+
+func (self *PublicKey) VerifySignature(digest [32]byte, sig []byte) error {
+	block, _ := pem.Decode(self.Value)
```

Comment:
is there always exactly one PK per region?

Reply:
yes. This may change if they rotate keys, but there is currently no plan to do that.